datahub-web-react/src/app/entity/shared/containers/profile/sidebar/Ownership/AddOwnersModal.tsx

```diff
@@ -76,49 +73,24 @@ export const AddOwnersModal = ({
     // Add owners Form
     const [form] = Form.useForm();
 
+    const inputEl = useRef(null);
+
+    useEffect(() => {
+        setTimeout(() => {
+            if (inputEl && inputEl.current) {
+                (inputEl.current as any).focus();
+            }
+        }, 1);
+    });
```

Comment:
curious why this is necessary if we have `autoFocus` set on the input?

Reply:
Removed and fixed it by unmounting the component.

```

            {showAddModal && (
                <AddOwnersModal
                    urn={mutationUrn}
                    defaultOwnerType={properties?.defaultOwnerType}
                    hideOwnerType={properties?.hideOwnerType || false}
                    type={entityType}
                    visible={showAddModal}
                    refetch={refetch}
                    onCloseModal={() => {
                        setShowAddModal(false);
                    }}
                />
            )}
```